tests/test_shapes.py

```diff
@@ -0,0 +1,75 @@
+import unittest
+from distutils.version import StrictVersion
+import onnx
+from sklearn.datasets import load_iris
+from sklearn.model_selection import train_test_split
+from sklearn.ensemble import RandomForestClassifier, RandomForestRegressor
+from skl2onnx import convert_sklearn
+from skl2onnx.common.data_types import FloatTensorType
+import onnxruntime as rt
+import numpy
+
+
+class TestSahpes(unittest.TestCase):
+
+    @unittest.skipIf(StrictVersion(onnx.__version__) < StrictVersion("1.6.0"),
+                     reason="not available")
+    @unittest.skipIf(StrictVersion(rt.__version__) < StrictVersion("1.0.0"),
+                     reason="not available")
+    def test_onnxruntime_shapes_reg(self):
+        iris = load_iris()
+        X, y = iris.data, iris.target
+        X_train, X_test, y_train, y_test = train_test_split(X, y)
```

Comment:
Can you use https://github.com/onnx/sklearn-onnx/blob/43e9c64380396fe71de7ab6dbb499cd52656134f/tests/test_utils/tests_helper.py#L51 for this? That would avoid code repetition.